daemon/web/.eslintrc.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Small thing, but I feel like there is some advantage to keeping this in the root directory alongside other general configuration (as it used to be) - unless this has to be in the same directory as package.json and stuff

Reply:
It would make sense to have all configuration/linting files in one place, but I'm still feeling a bit reluctant to do that. I generally promote files up a level it if needs to know of files from that directory. I'm also inclined to enforce a stronger separation between the web-app project and the daemon/cli. Also, [this](https://github.com/Microsoft/vscode-eslint/issues/196).